.devcontainer/Dockerfile

```diff
@@ -62,13 +64,12 @@ RUN sudo bash -c 'curl https://get.wasmer.io -sSfL | sudo WASMER_DIR=/home/vscod
 RUN sudo bash -c 'mkdir -p /home/vscode/.local/juvix'
 RUN sudo bash -c 'chmod -R a+rXw /home/vscode/.local/juvix'
 
-RUN sudo bash -c 'curl -s https://api.github.com/repos/anoma/juvix/releases/latest \
-      | grep -a "browser_download_url.*linux" \
-      | cut -d : -f 2,3 \
-      | tr -d \" \
-      | wget --output-document juvix.zip -qi - \
-      && unzip juvix.zip \
-      && mv juvix /home/vscode/.local/bin/juvix'
+
+RUN sudo bash -c 'curl --proto "=https" --tlsv1.2 -sSf https://raw.githubusercontent.com/anoma/juvix-installer/main/juvix-installer.sh \
+      | sudo JUVIX_INSTALLER_INSTALL_VAMPIR_YES=true JUVIX_INSTALLER_ASSUME_YES=true sh \
+      && sudo mv /root/.local/bin/juvix /home/vscode/.local/bin/ \
+      && sudo mv /root/.local/bin/vamp-ir /home/vscode/.local/bin/'
```

Comment:
Check if you can use `XDG_BIN_HOME=/home/vscode/.local/bin` to not use `mv` here.